detox/scripts/preboot_ios_simulator.sh

```
@@ -0,0 +1,232 @@
+#   - Drops the redundant 8–10s bootstatus poll after bootstatus already completed.
+#   - Brand-new simulators (simctl create) still use boot → shutdown → configure → boot.
+#
+# Requires: DEVICE_NAME, DEVICE_OS_VERSION. Writes SIMULATOR_ID to GITHUB_ENV when set.
+# Optional: PREBOOT_SKIP_PREWARM=1, PREBOOT_PREWARM_SECS (default 15).
+
+set -euo pipefail
+
+readonly BUNDLE_ID="com.mattermost.rnbeta"
+readonly AUTOFILL_MARKER="mattermost-ci-autofill-v1"
+readonly PREWARM_SECS="${PREBOOT_PREWARM_SECS:-15}"
+readonly REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)